### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**
Fixes misspellings identified by the [check-spelling action](https://github.com/marketplace/actions/check-spelling).

The misspellings have been reported at https://github.com/jsoref/AlmaLinux-wiki/actions/runs/5763920625#summary-15626763992

The action reports that the changes in this PR would make it happy: https://github.com/jsoref/AlmaLinux-wiki/actions/runs/5763920843#summary-15626764739

### CODE REVIEW

File: docs/documentation/building-packages-guide.md [hunk 1]
```diff
@@ -246,8 +246,8 @@ Finish: Outputting list of installed packages
 Start: rpmbuild zlib-1.2.11-17.el8.src.rpm
 ...
 patching file contrib/minizip/unzip.c
-Patch #2 (zlib-1.2.11-IBM-Z-hw-accelrated-deflate-s390x.patch):
-+ echo 'Patch #2 (zlib-1.2.11-IBM-Z-hw-accelrated-deflate-s390x.patch):'
+Patch #2 (zlib-1.2.11-IBM-Z-hw-accelerated-deflate-s390x.patch):
++ echo 'Patch #2 (zlib-1.2.11-IBM-Z-hw-accelerated-deflate-s390x.patch):'
```
Comment (jsoref) [inline docs/documentation/building-packages-guide.md:250]: If this is an example from a real package, I should probably drop this...

File: docs/LiveMedia.md [hunk 2]
```diff
@@ -66,8 +66,8 @@ For Windows OS there is a helpful free and open-source application - [Rufus](htt
 
 Open the application, choose your target USB, ISO you need to burn, press start - and Live OS is ready to run. 
 
-**MacOS:**
+**macOS:**
 
-The cross-platform tool [balenaEtcher](https://www.balena.io/etcher/) is used to write images on macOS. It is simple too. Open banlenaEtcher, choose the image and the USB, press Flash. 
+The cross-platform tool [balenaEtcher](https://www.balena.io/etcher/) is used to write images on macOS. It is simple too. Open balenaEtcher, choose the image and the USB, press Flash.
```
Comment (jsoref) [inline docs/LiveMedia.md:71]: brand

File: docs/Transparency.md [hunk 3]
```diff
@@ -77,7 +77,7 @@ The Board of Directors meets monthly on the second Tuesday of the month. At each
 
 ## Membership Committee
 
-In September of 2021, the Board of Directors [opened up membership applications](https://almalinux.org/blog/what-almalinux-foundation-membership-means-for-you/) for all the members of the AlmaLinux community wishing to participate in the the future of AlmaLinux. The Board also ratified the Membership Committee Governance, and appointed a selection of existing Foundation Members to participate in the Membership Committee. 
+In September of 2021, the Board of Directors [opened up membership applications](https://almalinux.org/blog/what-almalinux-foundation-membership-means-for-you/) for all the members of the AlmaLinux community wishing to participate in the future of AlmaLinux. The Board also ratified the Membership Committee Governance, and appointed a selection of existing Foundation Members to participate in the Membership Committee.
```
Comment (jsoref) [inline docs/Transparency.md:80]: not technically spelling, but spell checkers have been flagging duplicate words for decades

File: docs/documentation/building-packages-guide.md (hunk 1 above)
Comment (jonathanspw) [inline docs/documentation/building-packages-guide.md:250]:
```suggestion
Patch #2 (zlib-1.2.11-IBM-Z-hw-accelrated-deflate-s390x.patch):
+ echo 'Patch #2 (zlib-1.2.11-IBM-Z-hw-accelrated-deflate-s390x.patch):'
```

This is actual output so let's leave it as-is.

File: docs/documentation/building-packages-guide.md [hunk 4]
```diff
@@ -262,7 +262,7 @@ patching file deflate.c
 ...
 patching file deflate.c
 patching file test/infcover.c
-+ echo 'Patch #6 (zlib-1.2.11-IBM-Z-hw-accelrated-deflate-fix.patch):'
++ echo 'Patch #6 (zlib-1.2.11-IBM-Z-hw-accelerated-deflate-fix.patch):'
```
Comment (jonathanspw) [inline docs/documentation/building-packages-guide.md:265]:
```suggestion
+ echo 'Patch #6 (zlib-1.2.11-IBM-Z-hw-accelrated-deflate-fix.patch):'
```

More real output, let's leave it alone.

**Review by jonathanspw — CHANGES_REQUESTED** [review]

**Pull request merged**